app/scripts/modules/amazon/src/securityGroup/configure/createSecurityGroupIngress.html

```diff
@@ -95,6 +95,11 @@
         </p>
       </div>
     </div>
-    <additional-ip-rules security-group-details="securityGroupDetails"></additional-ip-rules>
+    <security-group-details-custom
+      security-group-details="securityGroup"
+      ctrl="ctrl"
+      scope="self"
+      application="application"
```

Comment:
I see that you're now storing `application` on the `$scope`.

Can we either: 
- Access `$scope.application` in the custom component and remove from the prop
- Remove the assignment of `application` onto `$scope`

Reply:
> I have some minor change requests, and then this looks good to me.

Thanks , made changes based on your feedback